test-resources/Inspections/IgnoreByNameSuppressorInspectionTest/nullable-list-field.skip.graphql.inc

```diff
@@ -1,3 +1,7 @@
 type User {
   alwaysTwoItems: [Int]!
 }
+
+input UserInput {
+  alwaysTwoItems: [Int]
```

Comment:
It worked like this due to introducing an additional parent `NonNullTypeNode` that bumps the relevant `InputValueDefinitionNode` into ancestors and thus causing it to be considered for suppression.

```graphql
  alwaysTwoItems: [Int]!
```